packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -171,3 +191,27 @@ class _TextScaleFactor extends StatelessWidget {
     );
   }
 }
+
+class _ThemedGoldenContainer extends StatelessWidget {
+  const _ThemedGoldenContainer({
+    this.brightness = Brightness.light,
+    required this.child,
+  });
+
+  final Brightness brightness;
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Theme(
+      data:
+          brightness == Brightness.light ? ThemeData.light() : ThemeData.dark(),
```

Comment:
I'm not sure this makes sense in any realistic use case. ThemeData.light() and ThemeData.dark() are the vanilla material themes.

If we were to use this in our app, it would show us what the widget under test would look like with the default material themes, not with our app's theme.

Reply:
Ack! I should have definitely known that. Well it was well intentioned!